frontends/ol-search-ui/src/components/ShareTooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useRef } from "react"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from "react-share"
+import { FacebookIcon, TwitterIcon, LinkedinIcon } from "react-share"
+import Tooltip from "rc-tooltip"
+
+type HelperProps = {
+  url: string
+  hideSocialButtons?: boolean | null | undefined
+  objectType?: string | null | undefined
+}
+
+export const ShareTooltipHelper: React.FC<HelperProps> = ({
+  url,
+  hideSocialButtons,
+  objectType
+}) => {
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const selectAndCopyLinkText = (e: React.MouseEvent<HTMLElement>) => {
+    e.preventDefault()
+    if (inputRef.current) {
+      navigator.clipboard.writeText(inputRef.current.value)
```

Comment:
Perhaps you discovered this, but apparently `navigator.clipboard` is only available on https websites, so this didn't work for me locally. Do you think that's a problem?

Can go back to the old execCommand way if need be.

Reply:
Weird, it worked fine for me at http://localhost:8063/

Reply:
Ah, that makes sense. [SO says](https://stackoverflow.com/a/51823007/2747370)
> This requires a secure origin — either HTTPS or localhost 

I'm running at http://cac.od.odl.local:8063/infinite/fields/philosophy/ ... I guess I don't need to do that, or maybe i can figure out how to tell chrome that cac.odl.local is a secure context